### PR TITLE
Release/0.6.2

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

File: src/main/scala/com/snowplowanalytics/iglu/server/Server.scala [hunk 1]
```diff
@@ -176,14 +176,14 @@ object Server {
         val action = migrate match {
           case Some(migration) =>
             migration.perform.transact(xa) *>
-              IO(logger.warn(s"All tables were migrated in $dbname from $migration"))
+              logger.warn(s"All tables were migrated in $dbname from $migration")
```
Comment (benjben) [inline src/main/scala/com/snowplowanalytics/iglu/server/Server.scala:179]: We don't care any more about the side effect ?

File: project/Dependencies.scala [hunk 2]
```diff
@@ -26,8 +26,8 @@ object Dependencies {
     val SchemaDdl  = "0.10.0-M7"
     val IgluClient = "0.6.1-M1"
```
Comment (dilyand) [inline project/Dependencies.scala:27]: Still `M`s?

File: src/main/scala/com/snowplowanalytics/iglu/server/Server.scala (hunk 1 above)
Comment (chuwy) [reply src/main/scala/com/snowplowanalytics/iglu/server/Server.scala:179]: Nope, we do care. But `logger` is delivered by log4cats and `logger.warn` already has `IO[Unit]` type, so our statement was `IO[IO[Unit]]`. Hence [the bug](https://github.com/snowplow-incubator/iglu-server/issues/42).

File: project/Dependencies.scala (hunk 2 above)
Comment (chuwy) [reply project/Dependencies.scala:27]: Yes, that's shame, but there were API changes in next `M`s that I don't fully remember. I afraid to break API in a patch release - many 3rd parties depend on those APIs.

File: src/main/scala/com/snowplowanalytics/iglu/server/Server.scala (hunk 1 above)
Comment (benjben) [reply src/main/scala/com/snowplowanalytics/iglu/server/Server.scala:179]: I see, great !

**Pull request merged**